CMakeLists.txt

```diff
@@ -3,18 +3,21 @@
 ## Configure the buddy-benchmark build.
 ##
 ##===----------------------------------------------------------------------===//
-
 cmake_minimum_required(VERSION 3.10)
 
 #-------------------------------------------------------------------------------
 # Project setup and globals
 #-------------------------------------------------------------------------------
 project(buddy-benchmark LANGUAGES CXX C)
 
-set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_STANDARD_REQUIRED YES)
-set(CMAKE_CXX_FLAGS "-no-pie")
-set(CMAKE_C_FLAGS "-no-pie")
+if(DEFINED DEEP_LEARNING_BENCHMARKS)
```

Comment:
Remove `DEEP_LEARNING_BENCHMARKS` condition and `no-pie`.

Use the following settings:

```
set(CMAKE_BUILD_TYPE Release)
set(CMAKE_CXX_STANDARD 17)
set(CMAKE_CXX_STANDARD_REQUIRED YES)
```